src/Workspaces/SharedUtilitiesAndExtensions/Workspace/VisualBasic/Extensions/ContextQuery/SyntaxTreeExtensions.vb

```diff
@@ -658,6 +658,42 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Extensions.ContextQuery
                                                        s_multilineStatementBlockStartKinds)
         End Function
 
+        <Extension>
```

Comment:
why did vb need this?

Reply:
I needed C#'s `IsInstanceContext` so I pulled it into the base context, and since VB had it exposed I wanted to implement this for correctness. Though there are no tests showing that it is correct. Do we want to keep that implementation as-is, since this does not affect anything else for now? The context is not a public API after all.

Reply:
> I needed C#'s IsInstanceContext so I pulled it into the base context

You know you're in c#, so you can cast the context to the derived type.

Reply:
> Though there are no tests showing that it is correct. 

Remove as vb has no need for this.

Reply:
Removed the VB implementation and made the fix C#-focused